.gitignore

```diff
@@ -5,6 +5,9 @@
 gae_runtime/*
 third_party/*
 core/templates/prod/*
+static/fonts/*
```

Comment:
What goes in static/fonts?

In general I prefer not to add anything at all to the existing .gitignore. Could you list the assets you need to find homes for, and I'll make suggestions for where they go? I'm thinking either a special folder in core/templates/prod/*, something in static/*, or something in third_party/* -- the first would contain generated files; the second would be part of this repo; the third would be untouched, downloaded third-party libs.